git/git.go

```diff
@@ -209,6 +209,18 @@ func DeleteLocalBranch(branch string) error {
 	return err
 }
 
+func DeleteRemoteBranch(branch string) error {
+	configCmd := GitCommand("push", "origin", "--delete", branch)
+	_, err := run.PrepareCmd(configCmd).Output()
+	return err
+}
+
+func DoesLocalBranchExist(branch string) bool {
+	configCmd := GitCommand("rev-parse", "--verify", branch)
+	_, err := run.PrepareCmd(configCmd).Output()
+	return err == nil
```

Comment:
I made the assumption here that any error means the branch doesn't exist. I figured that will be more stable in the future than doing a string match on the error message.